core/trino-main/src/main/resources/webapp/src/components/WorkerList.jsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+
+const SMALL_SPARKLINE_PROPERTIES = {
+    width: '100%',
+    height: '57px',
+    fillColor: '#3F4552',
+    lineColor: '#747F96',
+    spotColor: '#1EDCFF',
+    tooltipClassname: 'sparkline-tooltip',
+    disableHiddenCheck: true,
+};
+
+export class WorkerList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            workerInfo: null,
+            initialized: false,
+            ended: false,
+            workers: [{
+                workerId: String,
+                workerIp: String,
+                workerVersion: String,
+                coordinator: Boolean,
+                state: String
+            }]
+        };
+        this.refreshLoop = this.refreshLoop.bind(this);
+    }
+
+    refreshLoop() {
+        clearTimeout(this.timeoutId);
+        $.get('/ui/api/worker', function (workerInfo) {
```

Comment:
I would rename `workerInfo` to `workers` to signal that this is an array.

Reply:
Done.